spec/sections/copies.bs

```diff
@@ -46,15 +46,15 @@ dictionary GPUImageDataLayout {
 };
 </script>
 
-A {{GPUImageDataLayout}} is a layout of <dfn dfn>images</dfn> within some linear memory.
+An <dfn dfn>image</dfn> is a 1D or 2D array of [=texel blocks=] which are all of the same {{GPUTextureFormat}}.
```

Comment:
I think it would be more precise to say it's always 2D (technically that it's an array of rows, where each row is the same size, but that's hard to write clearly). The way copy parameters are worded (`bytesPerRow` and `rowsPerImage`), one texture(/texture copy region) is comprised of one or more images, each comprised of one or more rows, each comprised of one or more texel blocks.

Reply:
Yeah, it's hard to communicate all of that clearly, especially since some of the following paragraphs do touch on these concepts. I took another crack at it, though. PTAL!